pkg/shell/base_index.js

```diff
@@ -205,6 +205,12 @@ function Frames(index, setupIdleResetTimers) {
         if (new_frame) {
             list[component] = frame;
             document.getElementById("content").appendChild(frame);
+            // Set background to dark mode
+            if (cockpit.dark_mode && frame.contentDocument && frame.contentDocument.documentElement) {
+                // --pf-global--BackgroundColor--light-200
+                const dark_mode_background = '#0f1214';
+                frame.contentDocument.documentElement.style.background = dark_mode_background;
+            }
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17319-20221028-082204-6abb7d32-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)